core/src/main/java/org/apache/calcite/sql/validate/SqlAbstractConformance.java

```diff
@@ -111,4 +111,7 @@ public boolean allowQualifyingCommonColumn() {
     return SqlConformanceEnum.DEFAULT.allowQualifyingCommonColumn();
   }
 
+  public boolean allowAliasUnnestArrayColumns() {
+    return SqlConformanceEnum.DEFAULT.allowAliasUnnestArrayColumns();
+  }
```

Comment:
How about the `MultiSet` type.

Reply:
Could you elaborate here?

Reply:
I mean you `unnest` support `multiset` type which is also supported in your code. But the method name here does not represent that.

Reply:
Got it. Do you have a better name in mind? I picked array because it seems to be a little more common (at least for me). But I am OK with other names. Let me know.

Reply:
Just `allowAliasUnnestColumns` seems better, no need to specify the operand type, i think.